contrib/jinzhu/gorm/gorm_test.go

```diff
@@ -281,3 +281,24 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestContext(t *testing.T) {
+	sqltrace.Register("postgres", &pq.Driver{})
+	db, err := Open("postgres", "postgres://postgres:postgres@127.0.0.1:5432/postgres?sslmode=disable")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	t.Run("db with context", func(t *testing.T) {
```

Comment:
Just one word please
```suggestion
	t.Run("with", func(t *testing.T) {
```